masonry/src/properties/checkmark.rs

```diff
@@ -0,0 +1,104 @@
+// Copyright 2025 the Xilem Authors
+// SPDX-License-Identifier: Apache-2.0
+
+use std::any::TypeId;
+
+use crate::core::{Property, UpdateCtx};
+use crate::peniko::color::{AlphaColor, Srgb};
+
+/// The color of a checkbox's "check" icon.
+#[expect(missing_docs, reason = "obvious")]
+#[derive(Clone, Copy, Debug, PartialEq)]
+pub struct CheckmarkColor {
+    pub color: AlphaColor<Srgb>,
+}
+
+impl Property for CheckmarkColor {
+    fn static_default() -> &'static Self {
+        static DEFAULT: CheckmarkColor = CheckmarkColor {
+            color: AlphaColor::BLACK,
+        };
+        &DEFAULT
+    }
+}
+
+/// The color of a checkbox's "check" icon when disabled.
+#[derive(Clone, Copy, Debug, PartialEq)]
+pub struct DisabledCheckmarkColor(pub CheckmarkColor);
+
+impl Property for DisabledCheckmarkColor {
+    fn static_default() -> &'static Self {
+        static DEFAULT: DisabledCheckmarkColor = DisabledCheckmarkColor(CheckmarkColor {
+            color: AlphaColor::BLACK,
+        });
+        &DEFAULT
+    }
+}
+
+/// The with the stroke with which a checkbox's "check" icon is drawn.
+#[expect(missing_docs, reason = "obvious")]
+#[derive(Clone, Copy, Debug, PartialEq)]
+pub struct CheckmarkWidth {
```

Comment:
Perhaps rather `CheckmarkStrokeWidth`? This could be mistaken as the horizontal length of the checkmark.

Reply:
Probably, yeah.